docs/source/features/sheet_management/schema.rst

```diff
@@ -4,3 +4,234 @@
+               },
+               {
+                   "name": "label",
+                   "exist_if": "$labels",
+                   "type": "enum",
+                   "values": "$labels",
+               },
+           ],
+       },
+   }
+   Vector = pt.template("Vector", vector_template)
+   vector = Vector(coords=pt.float32(), labels=["cat", "dog"])
+
+
+**************
+ Schema Files
+**************
+
+Graviti SDK provides :func:`~graviti.portex.base.read_yaml` and :func:`~graviti.portex.base.read_json` to read the Portex type from a yaml or a json file.
+
+Take the `VOC2012Detection schema file`_ as an example.
+
+.. _VOC2012Detection schema file: https://github.com/Project-OpenBytes/standard/blob/main/example/VOC2012Detection.yaml
+
+.. code:: python
+
+   import graviti.portex as pt
+   schema = pt.read_yaml("VOC2012Detection.yaml")
+
+****************
+ Schema Package
+****************
+
+Graviti SDK supports to use exteranl packages defined under a repo. By giving the repo and revision, the package can be initialized and used locally.
+
+SDK provides :func:`~graviti.portex.builder.build_package` to build an exteranl Portex type package from the repo.
+
+Take `standard`_ as an example, which is used as the standard exteranl package by Graviti.
+
+.. _standard: https://github.com/Project-OpenBytes/standard
+
+.. code:: python
+
+   import graviti.portex as pt
+   ob = pt.build_package("https://github.com/Project-OpenBytes/standard", "main")
+   box2d = ob.label.Box2D(categories=["cat", "dog"])
+
+.. note::
+   Using branch as the revision to build the exteranl package is unstable, since the latest commit may change.
+
+   Tag or commit ID as revision is recommended.
+
+
+****************
+ Schema Methods
+****************
+
+PyArrow
+=======
+
+Graviti SDK provides methods to convert PyArrow schema to Portex schema and convert Portex schema to PyArrow schema.
+
+
+.. code:: python
+
+   import graviti.portex as pt
+   import pyarrow as pa
+   record = pt.record([("x", pt.int32()), ("y", pt.int32()), ("categories", pt.enum(values=["cat", "dog"]))])
+   # convert Portex Type to PyArrow builtin types
+   pa_struct = record.to_pyarrow()
+   # convert record type to PyArrow schema
+   pa_schema = pa.schema(record.fields.to_pyarrow())
+
+   # convert PyArrow type to Portex type
+   pa_type = pa.struct([pa.field("x", pa.float32()), pa.field("y", pa.float32())])
+   portex_type = pt.PortexType.from_pyarrow(pa_type)
+
+
+Expand
+======
+
+For better comprehension and operations, SDK provides methods to expand exteranl Portex type to builtin types.
+
+.. code:: python
+
+   import graviti.portex as pt
+   ob = pt.build_package("https://github.com/Project-OpenBytes/standard", "main")
+   box2d = ob.label.Box2D(categories=["cat", "dog"])
+   # Expand the first layer of the exteranl type
+   internal_type = box2d.internal_type
+   # Expand the top level of the exteranl type to internal type
+   builtin_type = box2d.to_builtin()
```

Comment:
do we need to print `internal_type` and `builtin_type` here for better understand?